packages/terra-date-input/src/DateInputField.jsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames/bind';
+import uniqueid from 'lodash.uniqueid';
+import IconError from 'terra-icon/lib/icon/IconError';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+
+import DateInput from './DateInput';
+import styles from './DateInputField.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The legend of the Date Input fieldset
+   */
+  legend: PropTypes.string.isRequired,
+  /**
+   * Name of the date input. The name should be unique.
+   */
+  name: PropTypes.string.isRequired,
+  /**
+   * Custom input attributes to apply to the day input
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  dayAttributes: PropTypes.object,
+  /**
+   * Whether the date input should be disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Can be used to set the display format. One of `month-day-year` or `day-month-year`.
+   * Overrides default locale display format.
+   */
+  displayFormat: PropTypes.string,
```

Comment:
Should this prop type be validated? 

```
PropTypes.oneOf(['month-day-year', 'day-month-year'])

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/838/commits/08c1db2e09a97f14f33f1cd64236a2d9940bf5a6